components/unique/Gallery/Item/AvailableActions.vue

```diff
@@ -311,3 +312,11 @@ export default class AvailableActions extends mixins(RmrkVersionMixin) {
   }
 }
 </script>
+
+<style scoped lang="scss">
+.only-border-top {
+  border-bottom: none;
+  border-left: none;
+  border-right: none;
+}
```

Comment:
I have seen this class name three times. 👀 

@roiLeo Do you have any idea about reusing this `top-border `effect?